app/src/main/java/fr/free/nrw/commons/upload/FailedUploadsFragment.kt

```diff
@@ -17,19 +17,16 @@ import fr.free.nrw.commons.profile.ProfileActivity
 import fr.free.nrw.commons.utils.DialogUtil
 import fr.free.nrw.commons.utils.ViewUtil
 import org.apache.commons.lang3.StringUtils
-import java.util.Locale
 import javax.inject.Inject
 
 /**
- * Fragment for displaying a list of failed uploads in Upload Progress Activity. This fragment provides
+ * Fragment for displaying a list of failed uploads in Upload Progress Activity. It provides
  * functionality for the user to retry or cancel failed uploads.
  */
 class FailedUploadsFragment :
     CommonsDaggerSupportFragment(),
     PendingUploadsContract.View,
     FailedUploadsAdapter.Callback {
-    @Inject
-    lateinit var pendingUploadsPresenter: PendingUploadsPresenter
```

Comment:
Any particular reason in removing this? I don't see any lint/warnings over this. I guess this patch will fix it.